pkgs/development/skaware-packages/skalibs/default.nix

```
@@ -21,6 +25,17 @@ buildPackage {
     # Empty the default path, which would be "/usr/bin:bin".
     # It would be set when PATH is empty. This hurts hermeticity.
     "--with-default-path="
+
+  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
```

skalibs only considers it to be cross compilation if the triples differ.  It'll happily run the tests if they match, even if the cross configure flags are set.

```suggestion
  ] ++ lib.optionals (stdenv.buildPlatform.config != stdenv.hostPlatform.config) [
```